api/health/client.go

```diff
@@ -22,18 +22,25 @@ func NewClient(uri string, requestTimeout time.Duration) *Client {
 }
 
 // GetLiveness returns a health check on the Avalanche node
-func (c *Client) GetLiveness() (*GetLivenessReply, error) {
-	res := &GetLivenessReply{}
+func (c *Client) GetLiveness() (*APIHealthReply, error) {
+	res := &APIHealthReply{}
 	err := c.requester.SendRequest("getLiveness", struct{}{}, res)
 	return res, err
 }
 
+// Health returns a health check on the Avalanche node
+func (c *Client) Health() (*APIHealthReply, error) {
```

Comment:
nit: since we are doing `getLiveness`, should we do `getHealth`?

Reply:
The intent is to deprecate `getLiveness`. Personally, I like `health.Health()` better than `health.getHealth()`